lib/AST/ConcreteDeclRef.cpp

```diff
@@ -59,6 +60,9 @@ void ConcreteDeclRef::dump(raw_ostream &os) {
     getSubstitutions().dump(os, SubstitutionMap::DumpStyle::Minimal);
     os << ']';
   }
+
+  os << " USR=";
+  ide::printDeclUSR(getDecl(), os);
```

Comment:
This was useful to figure out USRs of references since these USRs are used as symbols in quoted trees. However this is not strictly necessary, so I reverted the change.